apps/wistia-videos/src/functions/getVideos.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  Project,
+  ProjectReduced,
+  WistiaError,
+  WistiaVideo,
+  WistiaProject,
+} from '../components/helpers/types';
+
+const PROJECTS_PER_PAGE = 100;
+const VIDEOS_PER_PAGE = 500;
+
+const fetchProjectsJSON = async (
+  bearerToken: string,
+  page: number,
+  accumulated: WistiaProject[]
+): Promise<WistiaProject[]> => {
+  let projects = await (
+    await fetch(`https://api.wistia.com/v1/projects.json${page ? '?page=' + page : ''}`, {
+      headers: {
+        Authorization: `Bearer ${bearerToken}`,
+      },
+    })
+  ).json();
+  accumulated = [...accumulated, ...projects];
```

Comment:
Small thing that might not matter, but I'd consider appending new results as opposed to prepending them as you do here, in case there is ordering from the API that matters (e.g. some specified sort order).

Reply:
Not sure I follow. `accumulated` are the results from previous calls, while `projects` are the results from a newer call based on pagination. Wouldn't this pattern follow what you're describing?

Reply:
Yes, you are right. That's just me being dumb! 🤦